rtv/templates/rtv.cfg

```diff
@@ -82,6 +82,9 @@ oauth_scope = edit,history,identity,mysubreddits,privatemessages,read,report,sav
 ; See https://imgur.com/account/settings/apps to generate your own key.
 imgur_client_id = 93396265f59dec9
 
+; Open a new browser window instead of a new tab in existing instance
+force_new_browser_window = False
```

Comment:
This should be moved up, out of the ``OAuth Settings`` section and below ``;theme = molokai``.

Reply:
Oops yeah missed that in fact!